views/_includes/banner.pug

```diff
@@ -1,4 +1,7 @@
-mixin banner(text, variant, spanChar)
+mixin banner(variant, spanChar)
   div(class="banner banner--" + variant +"")
     span(class="banner__icon banner--"+ variant +"__icon" aria-hidden="true")= spanChar || 'i'
-    != text
\ No newline at end of file
+    if block
```

Comment:
Is this supposed to have a name like the block variables or etc or is this some other sort of magic?🔮

Reply:
it's magic 🐰 

https://pugjs.org/language/mixins.html#mixin-blocks

Reply:
There's another one here: https://github.com/cds-snc/cra-claim-tax-benefits/blob/master/views/_includes/buttonMixin.pug#L11